src/app/containers/ArticleTimestamp/timeFormatTests/expectedFormats.json

```diff
@@ -101,10 +101,10 @@
   },
   "gujarati": {
     "default": {
-      "one minute ago": "એક મિનિટ પેહલા",
-      "five minutes ago": "5 મિનિટ પેહલા",
-      "one hour ago": "એક કલાક પેહલા",
-      "five hours ago": "5 કલાક પેહલા",
+      "one minute ago": "એક મિનિટ પહેલા",
+      "five minutes ago": "5 મિનિટ પહેલા",
+      "one hour ago": "એક કલાક પહેલા",
+      "five hours ago": "5 કલાક પહેલા",
```

Comment:
should these have changed?

Reply:
https://github.com/moment/moment/blob/e1a969c8adadcb28ddec2fe382b4103fa5cf7abe/src/locale/gu.js#L63

It seems to have been recently updated a part of moment.

Reply:
@jroebu14 There was a update in moment: https://github.com/moment/moment/commit/3e1a95b5ef0a505fec567c5b2290fd80e1da18f9#diff-0bca40e277efcb9c8f6716725c521454R63